docs/guides/examples/cli/buckets_browse.sh

```diff
@@ -0,0 +1,23 @@
+reduct-cli alias add local -L http://localhost:8383 -t "my-token"
```

Comment:
Is it possible to seperate the CLI commands to make it easier to copy past?

Reply:
it is one file, if I split it then I the tests will be even more complicated. I'd like to keep it like this.